portal-impl/src/content/Language.properties

```diff
@@ -3883,6 +3883,7 @@ request-holiday=Request Holiday
 request-is-larger-than-x-and-could-not-be-processed=Request is larger than {0} and could not be processed.
 request-pending=Request Pending
 request-social-networking-summary-add-friend={0} wants to be your friend.
+request-social-networking-summary-add-connection={0} would like to add you as a connection.
```

Comment:
We shouldn't just copy this to `portal-impl` from `contacts-web`.  If it belongs here, we move all the translations here from `contacts-web`.